### ISSUE TITLE
keep parameters on single line, false positive

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#833]
following has 2 parameters, but an error is reported

```abap
IF NOT line_exists( structure-sub[ foo = bar
                                   boo = moo ] ).
```
